src/modules/trade/pure/UnlockWidgetScreen/index.tsx

```diff
@@ -51,9 +53,11 @@ export function UnlockWidgetScreen({
       )}
 
       <styledEl.ControlSection>
-        <span>
-          Learn more about <ExternalLink href={buttonLink}>{orderType} orders ↗</ExternalLink>
-        </span>
+        {showLink && (
+          <span>
+            Learn more about <ExternalLink href={buttonLink}>{orderType} orders ↗</ExternalLink>
```

Comment:
why not making `buttonLink` optional param instead?

Reply:
this way, we just need to not specify the URL until is available

Reply:
Good suggestion, will update.